### PR TITLE
#376 - IT added

### LINKED ISSUE #376: Subdirectories in rpm repository
Rpms in the rpm repository can be organized in subdirectories, tag `location` in the `primary.xml` should specify rpm file location relatively to the rpm repository, for example for file structure:
```
../
repodata/
subdir/
    rpm_in_subdir.rpm
rpm_not_in_subdir.rpm
```
`primary.xml` should contain `location` tags
```xml
<package type="rpm">
  <name>rpm_in_subdir</name>
  ...
  <location href="subdir/rpm_in_subdir.rpm"/>
  ...
</package>
<package type="rpm">
  <name>rpm_not_in_subdir</name>
  ...
  <location href="rpm_not_in_subdir.rpm"/>
  ...
</package>
```

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #376 
I've added one more rpm package and extended `RpmSliceITCase` to make sure packages from repository subdirs can be successfully installed. Also, added logging dependencies required by logger to write logs.

### CODE REVIEW

File: src/test/java/com/artipie/rpm/http/RpmSliceITCase.java [hunk 1]
```diff
@@ -61,6 +63,21 @@
...
+     */
+    private static final ListOf<String> INSTALLED = new ListOf<>(
+        "Installed", "aspell-12:0.60.6.1-9.el7.x86_64",
+        "time-1.7-45.el7.x86_64", "Complete!"
+    );
+
+    /**
+     * Packaged list verifier.
+     */
+    private static final ListOf<String> LIST = new ListOf<>(
```
Comment (olegmoz) [inline src/test/java/com/artipie/rpm/http/RpmSliceITCase.java:77]: @olenagerasimova I'd suggest to change this field's name to more informative one, such as `AVAILABLE`. This way difference with `INSTALLED` will be obvious

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, approved

**Pull request merged**